pkg/storage/cedar_system_bookmark_test.go

```diff
@@ -75,24 +73,15 @@ func (s *StoreTestSuite) TestFetchCedarSystemIsBookmarkedLOADER() {
 		})
 		s.NoError(err)
 
-		m := map[string]any{
-			"cedar_system_id": cedarSystemID,
-			"eua_user_id":     appcontext.Principal(ctx).ID(),
+		req := models.BookmarkRequest{
```

Comment:
I can't leave a comment on the specific line -- but `TestFetchCedarSystemIsBookmarkedByCedarSystemIDs` might be a better name for this test, now that the underlying function was renamed

Reply:
nice catch! thanks!